workflows/crisprseq_screening.nf

```diff
@@ -262,6 +262,12 @@ workflow CRISPRSEQ_SCREENING {
     ch_versions = ch_versions.mix(BAGEL2_GRAPH.out.versions)
     }
 
+    if(params.mle_control_sgrna) {
+        ch_mle_control_sgrna = Channel.fromPath(params.mle_control_sgrna)
+    } else {
+        ch_mle_control_sgrna = Channel.empty()
+    }
```

Comment:
Seeing the code now, I think we can actually add it to the `INITIALISATION_CHANNEL_CREATION_SCREENING` subworkflow :)